### DISCUSSION TITLE
Price checking to automatically be in Exalts and Divs instead of Chaos Orbs and Divs

### DISCUSSION THREAD

**Discussion by HerbLyfe** [Kvan7/Exiled-Exchange-2#144] [Ideas]
Title.

Currently you have to manually price check everything on the website for most items cause the trade website is fukt but could we be able to see the price of standard stock items (catalysts/tablet/other basic shit) in Exalts instead of Chaos orbs

**Comment by Kvan7** [comment]
This has been completed
